adhoc/node.yaml

```diff
@@ -103,7 +103,7 @@ services:
     volumes:
       - ${SAWOOTH_RAFT:-..}:/project/sawtooth-raft
     working_dir: /project/sawtooth-raft/
-    command: ./target/debug/raft-engine --connect tcp://validator:5050 -v
+    command: ./target/release/raft-engine --connect tcp://validator:5050 -v
```

Comment:
This change should be made in a separate commit (and possibly a separate PR, as it is less controversial than the logging change.